back-end/tests/routers_test/loansEndPoint.test.js

```diff
@@ -1,11 +1,19 @@
 const request = require('supertest')
 const app = require('../../app')
 const loanService = require('../../services/loansService')
+const commonAuthTest = require('./commonAuthTest')
+const {
+  noContentTypeError,
+  createLoanSuccess,
+  createLoanFail
+} = require('../MockData/loan_mockData')
 
 jest.mock('../../services/loansService')
 loanService.getLoansWithProductsByUserId.mockReturnValue({ loans: 'you have got a list of loans' })
 
 describe('GET /loans', () => {
+  commonAuthTest('get', '/loans')
```

Comment:
@Jeffzholy would this work: 
```suggestion
  commonAuthTest(request(app).get('/loans'))
```

Reply:
This doesn't work...